samples/cds-sample-application/package.json

```diff
@@ -25,26 +25,26 @@
     "ci-package": "rimraf deployment && mkdir deployment && cp -r dist package.json package-lock.json db srv .cdsrc.json deployment/",
     "cds-build": "cds build/all",
     "cds-deploy": "cds deploy",
-    "generate-client": "npx generate-odata-client --inputDir resources/service-specs --outputDir src/generated --clearOutputDir --generateJs=false --serviceMapping=resources/service-specs/service-mapping.json"
+    "generate-client": "npx generate-odata-client --input resources/service-specs --outputDir src/generated --clearOutputDir --transpile=false --optionsPerService=resources/service-specs/service-mapping.json"
   },
   "dependencies": {
     "@nestjs/common": "^8.4.7",
     "@nestjs/core": "^8.4.7",
     "@nestjs/platform-express": "^8.4.7",
-    "@sap/cds": "^6.1.3",
-    "@sap/cds-dk": "^6.5.2",
+    "@sap/cds": "6.1.3",
```

Comment:
Why are these dependencies pinned here?

Reply:
I had an issue with broken types in the `cds` dependency. I checked which was the installed version which was working and pinned it to this version.